README.md

```diff
@@ -27,30 +27,24 @@ $ python manage.py runserver
 
 ## Code Style
 
-This codebase uses the PEP8 code style. We enforces this with flake8. We use almost all the rules
-used in the Requests project.
-
-In addition to the standards outlined in PEP 8, we have a few guidelines:
+This codebase uses the PEP8 code style. We enforces this with isort, yapf & flake8.
```

Comment:
`We enforce` :)

Reply:
Guess I should read the text i change.. :upside_down_face: